README.md

```diff
@@ -71,17 +69,18 @@ additional asynchronous work.
 1. When the `Outputer` finishes its end of _event_ work, the `Lane` is considered to be done with that _event_ and the cycle repeats.
 
 ## Running tests
-The `threaded_io_test` takes up to 5 command line arguments
+The `threaded_io_test` takes up the following command line arguments
 ```
-threaded_io_test <Source configuration> [# threads[/useIMT]] [# conconcurrent events] [time scale factor] [max # events] [<Outputer configuration>]
+threaded_io_test -s <Source configuration> [-t <# threads>] [--use-IMT=<T/F>] [-l <# conconcurrent events>] [-s <time scale factor>] [ -n<max # events>] [-o <Outputer configuration>]
 ```
 
-1. `<Source configuration>` : which `Source` to use and any additional information needed to configure it. Options are described below.
-1. `[# threads[/useIMT]]` : number of threads to use in the job. Optionally can add '/useIMT' directly after the number of threads to turn on ROOT's implicit multithreaded (IMT). 
-1. `[# concurrent events]` : number of _events_ (that is `Lane`s) to use. Best if number of events is less than  or equal to number of threads.
-1. `[time scale factor]` : used to convert the property of the _event_ data products into microseconds used for the sleep call. A value of 0 means no sleeping. A value less than 0 prohibits the creation of the objects which do the sleep.
-1. `[<Outputer configuration>]` : optional, used to specify which `Outputer` to use and any additional information needed to configure it. The exact options are
-described below.
+1. `--source, -s` `<Source configuration>` : which `Source` to use and any additional information needed to configure it. Options are described below.
+1. `--num-threads, -t` `<# threads>` : number of threads to use in the job. 
+1. `--use-IMT` turn on or off ROOT's implicit multithreaded (IMT). Default is off.
+1. `--num-lanes, -l` `<# concurrent events>` : number of concurrent _events_ (that is `Lane`s) to use. Best if number of events is less than  or equal to number of threads. Default is values used for `--num-threads`.
+1. `--scale, -s` `<time scale factor>` : used to convert the property of the _event_ data products into microseconds used for the sleep call. A value of 0 means no sleeping. A value less than 0 prohibits the creation of the objects which do the sleep. Default is 0.
```

Comment:
is it okay to use `-s` for both scale and source? I see the full option names are different.

Reply:
fixed